### ISSUE TITLE
[Feature Request] Allow running in the background

### ISSUE THREAD

**Issue by reisaraujo-miguel** [Betterbird/thunderbird-patches#71]
Something that I feel is a must have feature for email clients, is the ability to run in the background to check for new emails.

The only way you can do this on Thunderbird is by ticking **"When Thunderbird is minimized move to tray"** in the settings and installing the **"Minimize on close"** extension. The only problem is that the **"When Thunderbird is minimized move to tray"** option is only available on Windows.

**Comment by Betterbird** [comment]
This was previously filed at issue 33.

Since BB is using the storage location of the TB profiles, this https://github.com/gyunaev/birdtray is apparently working.

**Comment by Leuug** [comment]
It kinda works, but Birdtray is full of bugs. And even if it wasn't, it's still better to have this functionality built-in than to depend on another program. 

It is not even necessary to have a tray icon, the application just need to be able to run in the background. Geary works this way, it runs on the background but doesn't have a tray icon. I have been using it just because of this feature.

**Comment by pflanzenandi** [comment]
Also missing a tray icon feature badly on linux.

**Comment by Betterbird** [comment]
The tray icon is issue #20, see https://github.com/Betterbird/thunderbird-patches/issues/20#issuecomment-1141040721.

**Comment by Siggi0904** [comment]
I'm using the included minimize to tray option from thunderbid/betterird at my windows 11 22H2 64 OS.

If I reopen Thunderbird/Betterbird from tray, the fullsize windows get lost and the window isn't maximized anymore.
Can anyone confirm that?

**Comment by Betterbird** [comment]
Hmm, this issue is about something else, you're talking about this bug:
https://bugzilla.mozilla.org/show_bug.cgi?id=1779548

**Comment by Siggi0904** [comment]
Hi, I don't know about these official bug.
But yes, sounds like the same.

What shall I do? Wait for fix at thunderbird or could it be fixed in betterbird?

**Comment by Betterbird** [comment]
I tried to reproduce it: Switched on "minimise to tray" (which I don't use usually). It always restores correctly to the previous size. Hard to fix something I can't reproduce.

**Comment by Siggi0904** [comment]
from maximized to maximized?
Sounds strange.

**Comment by Betterbird** [comment]
Yes, from maximised to maximised (and non-maximised to non-maximised). I guess the TB folks haven't done anything about bug [1779548](https://bugzilla.mozilla.org/show_bug.cgi?id=1779548) because they can't reproduce it either.

**Comment by Zahrun** [comment]
In reply to https://github.com/Betterbird/thunderbird-patches/issues/20#issuecomment-1383105403
Yes, I actually got to know about Betterbird when I was looking for good alternatives for Thunderbird closing to tray on linux. MinimizeToTray Reanimated has version that works with Thunderbird 60.0 - 63.*, I think it is not maintained anymore. I used to use Birdtray, but had some issues with it. I am currently using SysTray-X, and it is fine on X11 but does not work on Wayland. Since now we have a tray icon that is working properly on Linux systems, I wish that Betterbird could support close to tray. It seems more attainable now. 

> I would love to have a "close to tray" option, and even nicer that a click on the tray icon would minimize to tray if betterbird is open and show betterbird if it was closed.
